actors/common/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchManagementActor.java

```diff
@@ -130,11 +134,39 @@ private void createCourseBatch(Request actorMessage) {
     Map<String, String> rollUp = new HashMap<>();
     rollUp.put("l1", (String) request.get(JsonKey.COURSE_ID));
     TelemetryUtil.addTargetObjectRollUp(rollUp, targetObject);
+
+    if (courseNotificationActive()) {
+
+      batchOperationNotifier(courseBatch, null);
+    }
+  }
+
+  private boolean courseNotificationActive() {
+    return Boolean.parseBoolean(
+        PropertiesCache.getInstance()
+            .getProperty(JsonKey.SUNBIRD_COURSE_BATCH_NOTIFICATIONS_ACTIVE));
+  }
+
+  private void batchOperationNotifier(
+      CourseBatch courseBatch, Map<String, Object> participantMentorMap) {
+    Request batchNotification = new Request();
+    batchNotification.setOperation(ActorOperations.COURSE_BATCH_NOTIFICATION.getValue());
+    Map<String, Object> batchNotificationMap = new HashMap<>();
+    if (participantMentorMap != null) {
```

Comment:
Line 155-160 is not as per our discussion yesterday. Was expecting the notification map to contain ADDED_MENTORS, REMOVED_MENTORS, ADDED_PARTICIPANTS, REMOVED_PARTICIPANTS

Reply:
Hello Sir, 
Actually it contains all four type of data but it is inside participantMentorMap. which is in notificationMap

Reply:
Doesn't have to be nested within a map. Also, ensure that the type of these are list of strings (user IDs).